server/README.md

```diff
@@ -0,0 +1,22 @@
+# TACC Frontera Web Portal - Server
```

Comment:
__Explanation__: I found this directory baffling, at first. I hope this document saves someone at least one headache.